core/templatetags/discover_uni_tags.py

```diff
@@ -59,17 +59,48 @@ def map_distance_learning_values(key, language):
 
 
 @register.simple_tag
-def should_show_accordion(course, accordion_type):
+def should_show_accordion(courses, accordion_type):
```

Comment:
Perhaps may be able to simplify this at some stage by using a lookup table to get the property name from the accordion_type and then using something like:

`if getattr(course, 'looked up prop name here')` and `return getattr(courses, 'looked up prop name here')`

Reply:
@philratcliffe is this the sort of thing you were suggesting?